test/realmodel/index.html

```diff
@@ -44,15 +44,50 @@
         a.dispatchEvent(e)
       }
 
+      async function downloadFirstResnet(data, filename) {
```

Comment:
@shenx3x FirstResnet and next SecondResnet  aren't definite, please use Resnet50v1 and Resnet50v2 to replace them.